src/App.tsx

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer
+    .filter(({ title, description }) => (
+      title
+        .toLowerCase()
+        .includes(query.toLowerCase())
+      || description
+        .toLowerCase()
+        .includes(query.toLowerCase())
+    ));
```

Comment:
You can make small separate function to compare two string values in lowerCase :)

Reply:
Why?)

Reply:
Because you kind of duplicate your logic twice(do the same but once on `title`, then on `description`. And better use function for it.